### PR TITLE
Update Elehide.txt

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/pull/88218#issuecomment-879669842

### CODE REVIEW

**Comment by Alex-302** [comment]
Too wide.

**Comment by piquark6046** [comment]
@Alex-302 Please re-review.

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -4347,6 +4347,8 @@ dap-news.com##.ads
 !------------------------------ Korean -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/pull/88347
+bosa.co.kr,econovill.com,sisafocus.co.kr,newsian.co.kr,idsn.co.kr,ggilbo.com,andongdaily.com,tiptipnews.co.kr,fntoday.co.kr,newstown.co.kr,womennews.co.kr,sisajournal.com,zum.com,safetynews.co.kr,koreabiomed.com,mindpost.or.kr,energydaily.co.kr,docdocdoc.co.kr,edaily.co.kr##div.banner_box
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:4351]: Wry previous rule is bad?
Checked randomly docdocdoc.co.kr - there are no `.banner_box` elements.

Comment (piquark6046) [reply EnglishFilter/sections/foreign.txt:4351]: scress
wesdf
https://www.docdocdoc.co.kr/news/articleView.html?idxno=2012593
https://www.docdocdoc.co.kr/
No.   There are exist.

Comment (piquark6046) [reply EnglishFilter/sections/foreign.txt:4351]: I think `[...] ##div.banner_box` is more safer.

**Pull request closed without merging**